.github/workflows/testsuite.yml

```diff
@@ -373,7 +373,7 @@ jobs:
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           cd win32
-          nmake CCTYPE=MSVC142 test
+          nmake CCTYPE=MSVC142 CFG=Debug test
```

Comment:
The CFG=Debug is needed in the Build entry too, for both jobs.

Sorry I missed this the first time around.